holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1219 @@
+        # Write the python dictionary version
+        python_path = link_path.rpartition('.')[0] + '.py'
+        name = os.path.basename(python_path)
+        parts = name.split('_')
+        name = '_'.join(parts[:2]) + '_links'
+        keys = list(interior_dict.keys())
+        keys.sort()
+
+        with open(python_path, 'w', encoding='latin-1') as f:
+            f.write(name + ' = {\n')
+            for valtype in (list, str):
+              for key in keys:
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)